src/composables/multisigAccounts.ts

```diff
@@ -148,10 +182,10 @@ export function useMultisigAccounts({ store }: IDefaultComposableOptions) {
 
             const hasPendingTransaction = !!txHash && !consensus.expired;
 
-            const consensusLabel = (
-              `${hasPendingTransaction ? consensus?.confirmedBy?.length : 0}`
-              + `/${consensus.confirmationsRequired} ${i18n.t('common.of')} `
-              + `${signers.decodedResult?.length}`
+            const consensusLabel = prepareConsensusLabel(
+              hasPendingTransaction ? consensus?.confirmedBy?.length : 0,
+              consensus.confirmationsRequired,
+                signers.decodedResult?.length,
```

Comment:
From the beginning this should not be placed here as it saves the outcome of the translation function to the object property. So when user changes the language this label won't change in some places.
So the `prepareConsensusLabel` function should be used directly in the templates.

Reply:
I'll create a task for it as it already existed in our current code